### PR TITLE
Analysis Detection SQL/DB Generation & Population

### PR DESCRIPTION
**Pull request by lucas-m98**

### CODE REVIEW

File: packet_analysis/sql/dao/arp.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from scapy.all import Packet
+from ..dao.sqlObject import sqlObject
```
Comment (briweinstein) [inline packet_analysis/sql/dao/arp.py:2]: Absolute paths here would be better, it's how we're doing it elsewhere

File: packet_analysis/sql/dao/arp.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from ..dao.ethernet import Ethernet, table_sql as ethernet_table_sql
+
+def table_sql() -> str:
```
Comment (briweinstein) [inline packet_analysis/sql/dao/arp.py:5]: What's the point of this function if it just returns the output of another function?

File: packet_analysis/sql/dao/dhcp.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+from scapy.all import Packet
+from ..dao.sqlObject import sqlObject
```
Comment (briweinstein) [inline packet_analysis/sql/dao/dhcp.py:2]: Absolute paths

File: packet_analysis/sql/dao/arp.py (hunk 2 above)
Comment (lucas-m98) [reply packet_analysis/sql/dao/arp.py:5]: Organizationally it makes sense to be identical to all of the other objects.

**Pull request merged**